test/unittest/blas3/blas3_gemm_batched_test.cpp

```diff
@@ -34,8 +34,8 @@ const auto BetaNonZeroLDMatch =
                        ::testing::Values(63, 128),   // k
                        ::testing::Values('n', 't'),  // transa
                        ::testing::Values('n', 't'),  // transb
-                       ::testing::Values(1.0),       // alpha
-                       ::testing::Values(1.0),       // beta
+                       ::testing::Values(3.0),       // alpha
+                       ::testing::Values(7.0),       // beta
```

Comment:
Why this change? There is no special case for alpha=1 or for beta=1 as far as I know.

Reply:
The issue this PR is fixing doesn't get triggered when `beta == 1`. I discovered that because I copy-pasted the original tests :) I thought it might be better in the future to have more complicated values to potentially capture more issues. I can remove it if you don't think that should be included in this PR.

Reply:
Right, I don't mind if it stays.